src/ripple/rpc/handlers/Fee1.cpp

```diff
@@ -30,6 +30,7 @@ namespace ripple
     {
         // Bail if fee escalation is not enabled.
         auto const view = context.app.openLedger().current();
+        BOOST_ASSERT(view);
```

Comment:
A change isn't required but the current ledger and fee escalation feature are checked here and redundantly in the immediate following call to `doRPC`. You might consider cleaning that up but I'll leave it to your discretion.

Reply:
Belt and suspenders. :grinning:  I've got plans to replace all of the amendment checks soon, so this'll be moot.